play/PlayerRules/CharacterCreation/05_ArmourList.txt

```diff
@@ -88,12 +71,67 @@ this by providing 'Hard Points'. Hard Points are special joints and fittings tha
 make it easy to attach equipment. When you purchase equipment, it will list how
 many Hard Points it consumes. You can't attach more equipment than you have available
 Hard Point space.
+	Hard Point Equipment/Shields/Cloaking often come in two sizes, "Primary Modules" and 
+"Secondary Modules". After you purchase and install a Primary Module, you can also purchase
+and install a Secondary Module(s) that augments the benefits of the Primary Module. The 
+benefits are listed in the description of a given module. They do not mix and match,
+Secondary Modules only work for the Primary Module they are designed for.
+	Some modules have a "____ Skill Requirement". In order to operate the module, you
+must have the required amount of the given skill.
+
+
+    =====================================
+        EVA SUITS AND FLIGHT HELMETS
+    =====================================
+These additions may be worn with or added to armour. Environment 
+(EVA) suits are air- and water-tight when combined with an EVA or flight 
+helmet. EVA suits are bulky, DEX skill checks have a -10% penalty and movement
+is reduced in RP. If the suit is hit more times than its "Suit Integrity"
+rating, it will no longer be able to maintain air pressure (player health 
+has to be damaged in order for "Suit Integrity" to count a hit). Suits 
+traditionally come in white, yellow or red.

```

Comment:
I love that this is finally made explicit and I  feel that you struck a very good balance with acknowledging that if you shoot a space suit too many times, it stops protecting you from space while at the same time not making it too complex.